ui/app/serializers/clients/monthly.js

```diff
@@ -1,11 +1,11 @@
-import { formatISO } from 'date-fns';
 import ApplicationSerializer from '../application';
+import { formatISO } from 'date-fns';
 
 export default ApplicationSerializer.extend({
   flattenDataset(payload) {
-    let topTen = payload.slice(0, 10);
+    let topTen = payload?.slice(0, 10);
```

Comment:
@hashishaw added the try `?` here and on line 8 - thoughts? the serializer may change a bit as we're still not sure of whether or not mounts will be included and displayed on the frontend. that's TBD, but wanted a catch in case there was an issue with the `by_namespace` payload. thought it should always be an array (just an empty one..)

Reply:
I would instead do something like this:
```
let topTen = payload ? payload.slice(0, 10); [];
```
Because as you have it, it will return null rather than the type which you expect, an array. Then you can remove the `?` from line 8 because it will always be an array. 
There's also a nit about styling, since the ternary `?.` operator is typically only used to check for keys _before_ you use the key for something. Hope that makes sense!